src/features/dashboard/ui/MultiplePieCharts.tsx

```
@@ -0,0 +1,59 @@
+    });
+
+    return Object.entries(countMap).map(([name, value]) => ({ name, value }));
+  };
+
+  if (responses.length === 0) {
+    return <p className="text-center text-gray-500 py-8">응답 데이터가 없습니다.</p>;
+  }
+
+  return (
+    <div className="grid grid-cols-1 gap-6 p-4">
+      {responses.map((option, _index) => {
```

ESLint

[error] '_index' is defined but never used.

(@typescript-eslint/no-unused-vars)